tasmota/tasmota_configurations.h

```diff
@@ -199,6 +199,7 @@
 #define USE_MCP39F501                            // Add support for MCP39F501 Energy monitor as used in Shelly 2 (+3k1 code)
 #define USE_SDM72                                // Add support for Eastron SDM72-Modbus energy monitor (+0k3 code)
 #define USE_SDM120                               // Add support for Eastron SDM120-Modbus energy monitor (+1k1 code)
+#define USE_SDM230                               // Add support for Eastron SDM230-Modbus energy monitor (+?? code)
```

Comment:
Uncomment as there is no room for additional sensors

Reply:
uncommented